control-plane/typeSpecConverter.cpp

```diff
@@ -111,6 +115,50 @@ bool TypeSpecConverter::preorder(const IR::Type_Name* type) {
     return false;
 }
 
+// This function is invoked if an architecure's model .p4 file has a Newtype.
```

Comment:
That doesn't make any sense to me, your comment should at least explain why this is the case
And if this is a bug (looks like it), add a TODO

Reply:
I didn't get the time to look into this one.  I added a TODO.